channels/lt.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U x-tvg-url="https://iptvx.one/epg/epg.xml.gz"
+#EXTINF:-1 tvg-logo="https://i.imgur.com/IRSgGY8.png",PINGVIN LOLO
+http://82.193.71.93/LoLoLT/tracks-v1a1/index.m3u8
```

Comment:
It's not working at the moment.
<img width="986" alt="Screenshot 2019-11-03 at 11 39 07" src="https://user-images.githubusercontent.com/7253922/68082492-939cfa80-fe2e-11e9-840e-080887fd1114.png">